app/views/api/v2/smart_class_parameters/main.json.rabl

```diff
@@ -2,5 +2,5 @@ object @smart_class_parameter
 
 extends "api/v2/smart_class_parameters/base"
 
-attributes :description, :override, :parameter_type, :default_value, :use_puppet_default, :required, :validator_type, :validator_rule,
+attributes :description, :override, :parameter_type, :default_value, :hidden_value, :use_puppet_default, :required, :validator_type, :validator_rule,
```

Comment:
I agree but I didn't want to remove is from global parameters, I'll replace with `hidden_value?` here and add it to global parameters